concordia/settings_dev.py

```diff
@@ -45,3 +45,4 @@
     ("concordia.utils", "get_anonymous_user"),
     ("concordia.models", "TranscriptionStatus"),
 ]
+S3_BUCKET_NAME = "crowd-dev-content"
```

Comment:
Unless you put the other setting(s) such as MEDIA_URL, this should not be here.

Reply:
Thanks removed it.